Dockerfile

```diff
@@ -239,8 +239,7 @@ COPY images/networkd.tar /rootfs/usr/images/
 # symlinks to avoid accidentally cleaning them up.
 COPY ./hack/cleanup.sh /toolchain/bin/cleanup.sh
 RUN cleanup.sh /rootfs
-COPY hack/containerd.toml /etc/containerd.toml
-COPY hack/containerd.toml /etc/containerd-system.toml
+COPY hack/containerd.toml /rootfs/etc/containerd/cri.toml
```

Comment:
Come to find out, we haven't been writing this file to the correct path so the config was never used. Things have been working fine without it so I don't think we need to configure the system containerd in any special way. This changes it so that we only configure the CRI containerd differently.

Reply:
I think because we had reaper before system/cri containerd split

Reply:
Exactly